components/console_cmd_mqtt/examples/ssl_mutual_auth/README.md

```diff
@@ -0,0 +1,139 @@
+| Supported Targets | ESP32 | ESP32-C2 | ESP32-C3 | ESP32-C6 | ESP32-H2 | ESP32-P4 | ESP32-S2 | ESP32-S3 |
+| ----------------- | ----- | -------- | -------- | -------- | -------- | -------- | -------- | -------- |
+
+# ESP-MQTT SSL Sample application (mutual authentication)
+
+This example illustrates the use of the MQTT command component for both secured and unsecured MQTT brokers.
+It connects to the broker at test.mosquitto.org using either unsecured transport or SSL transport with a client certificate. The operations of subscribing to and unsubscribing from a specified topic, as well as sending a message on that topic, can be performed using commands.
+(Please note that the public broker is maintained by the community so may not be always available, for details please visit http://test.mosquitto.org)
+
+It uses ESP-MQTT library which implements mqtt client to connect to mqtt broker.
+
+## How to use example
+
+### Hardware Required
+
+This example can be executed on any ESP32 board, the only required interface is WiFi and connection to internet.
+
+### Configure the project
+
+* Open the project configuration menu (`idf.py menuconfig`)
+* Configure Wi-Fi or Ethernet under "Example Connection Configuration" menu. See "Establishing Wi-Fi or Ethernet Connection" section in [examples/protocols/README.md](../../README.md) for more details.
+
+* Generate your client keys and certificate
```

Comment:
The steps here are for testing with one specific broker and we should state that.

Reply:
Done